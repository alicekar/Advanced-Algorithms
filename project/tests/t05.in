8
1111011010111101
0010011100011101
0010011110101011
1001010000101000
1010000000111001
0100011010111010
1111001110100110
1100111100001101
12
