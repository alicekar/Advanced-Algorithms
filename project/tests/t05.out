1 2
1 5
1 7
2 3
2 8
3 6
4 5
